doc/api/n-api.md

```diff
@@ -4542,7 +4542,7 @@ NAPI_EXTERN napi_status napi_get_uv_event_loop(napi_env env,
 <!--lint disable no-unused-definitions remark-lint-->
 ## Asynchronous Thread-safe Function Calls
 
-> Stability: 1 - Experimental
+> Stability: 2 - Stable
```

Comment:
Since N-API itself is stable, there's no need to mark any parts of it as stable. Simply remove the Stability indicator instead.

Refs: https://github.com/nodejs/node/pull/28485

@cjihrig

Reply:
> Since N-API itself is stable, there's no need to mark any parts of it as stable. Simply remove the Stability indicator instead.
> 
> Refs: #28485
> 
> @cjihrig

Oh, wait, am I wrong about it in this particular case? I see @cjihrig already approved this change.

Reply:
No, you're correct. If a top level module is marked as stable, there is no need to explicitly call that out for individual APIs. Good catch.

Reply:
@Trott @cjihrig Actually, N-API is more granular than that. It is versioned such that each N-API version is semver-minor wrt. the last. We also have a practice where N-APIs added after version *x* are added to a pool of experimental N-APIs. Occasionally, based on community feedback and on whether it is possible to backport to all LTS versions, we gather a portion of the N-APIs from the pool of those marked experimental and release them as N-API version *x+1*. This coincides with a Node.js minor release.

Thus, portions of N-API may be experimental, whereas N-API version *x* is not.

The thread-safe function was marked as stable when N-API version 4 was released. At that time, all thread-safe function N-APIs were marked as stable, however, the overall heading (addressed in this PR) was forgotten.

An example of a set of N-APIs that will likely remain experimental until the end of v8.x is bignum. There is no engine support for bignum in v8.x, so we cannot backport those N-APIs to v8.x. While we have made no promise that, when we release a new version of N-API, it will be available on all current LTS versions, we do fairly strongly endeavour to make new N-APIs available across all current LTS versions.

Reply:
... as we mark them stable.

Reply:
@gabrielschulhof using this PR as an example - is there any reason to call out thread-safe functions as explicitly stable? As a user looking at the N-API docs I would assume that the thread-safe functions are stable unless specifically called out as experimental, deprecated, etc.

Reply:
@cjihrig OK, I see. The designation disappears when it becomes stable and the designation of larger scope applies 👍